model/configuration.py

```diff
@@ -54,6 +56,30 @@ class ExternalIntegrationLink(Base, HasFullTableCache):
     )
     purpose = Column(Unicode, index=True)
 
+    @classmethod
+    def _get_settings(cls):
```

Comment:
Is this what you meant for https://github.com/NYPL-Simplified/circulation/pull/1333/files#r333120505 ?

Reply:
Yes, but `_get_settings` is a generic name and this doesn't need to be a function. You can make a list of COLLECTION_MIRROR_SETTINGS and then use it from controller code.